pkgs/tools/archivers/sharutils/default.nix

```diff
@@ -25,6 +25,10 @@ stdenv.mkDerivation rec {
       url = "https://sources.debian.org/data/main/s/sharutils/1:4.15.2-2+deb9u1/debian/patches/01-fix-heap-buffer-overflow-cve-2018-1000097.patch";
       sha256 = "19g0sxc8g79aj5gd5idz5409311253jf2q8wqkasf0handdvsbxx";
     })
+    (fetchurl {
```

Comment:
fetchpatch?

Reply:
Same as https://github.com/NixOS/nixpkgs/pull/66528#discussion_r374265548 (so that the same discussion isn't split to two places)